lib/provider.js

```diff
@@ -253,7 +252,11 @@ const parseOutput = (text) => {
   text = text && text.slice(0, text.indexOf('\n'));
 
   if (!text) {
-    throw noOutputFromOracleError();
+    atom.notifications.addWarning('Elm Autocomplete Error', {
```

Comment:
Hrm, I'm not so sure about this change. I'd like to keep throwing errors at this point so we can continue to catch problems from different users and fix them at their source, rather than simply hiding the fact that there is an error. Good news! You've fixed the root cause of the problem above :) So I'd prefer to revert this part of the change. Thoughts?